chef_master/source/config_rb_server_optional_settings.rst

```diff
@@ -305,7 +337,9 @@ This configuration file has the following settings for ``keepalived``:
    Specify that a lower priority machine maintains the master role, even if a higher priority machine is available. (This setting configures the ``noprempt`` value in VRRP.) Default value: ``'true'``.
 
 ``keepalived['vrrp_instance_password']``
-   The secret key for VRRP pairs. This value is generated randomly when the bootstrap server is installed and does not need to be set explicitly. Default value: ``'sneakybeaky'``.
+   Legacy configuration for the secret key of VRRP pairs. This value is generated randomly when the bootstrap server is installed and does not need to be set explicitly. Default value: **generated**.
+
+   To override this, use the :ref:`ctl_chef_server_secrets_management` commands: ``chef-server-ctl set-secret keepalived vrrp_instance_password``.
```

Comment:
Do we want to encourage them to use this override (as we did with `data_collector['token']`)?